scripts/merge_tables.R

```diff
@@ -73,7 +78,18 @@ option_list <- list(
 )
 
 # Parse command-line arguments
-opt_parser <- OptionParser(usage=paste("Usage:", script, "[OPTIONS] --input_dir <path/to/input/files>\n", sep=" "), option_list = option_list, add_help_option=FALSE, description=msg)
+opt_parser <-
+  OptionParser(
+    usage = paste(
+      "Usage:",
+      script,
+      "[OPTIONS] --input_dir <path/to/input/files>\n",
+      sep = " "
+    ),
+    option_list = option_list,
+    add_help_option = FALSE,
+    description = msg
+  )
```

Comment:
I'm not sure which linter you have used? I'm fine with all the line wraps, but there are some issues. For example, I am pretty sure it's convention that there is no white space between function args, the `=` and the value, i.e., it should be `arg=value`, not `arg = value`.

Some more comments on this later, probably.

Reply:
I used the code format  tool within Rstudio. Now I used [lintr](https://lintr.r-lib.org/) with the default style `"tidyverse"`, and in fact, it made me put spaces between the `=` and whatever there was before and after...

Reply:
Fine with me. Doesn't matter if I like it as long as things are consistent :) But wondering why it was wrapping lines in different styles. Anyway, not worth investigating.